compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/AbstractTokeReader.java

```diff
@@ -24,7 +24,8 @@
  *
  * @since 1.3.0
  */
-public abstract class AbstractNodeSupplier {
+// TODO can we make this an interface?
+public abstract class AbstractTokeReader {
```

Comment:
Does `AbstractTokeReader` need to be changed as `AbstractTokenReader`?

Reply:
Supun fixed it in his next PR